WordPress/Classes/ViewRelated/Reader/ReaderHelpers.swift

```diff
@@ -578,19 +578,19 @@ extension ReaderHelpers {
                 return true
             }
 
-            // first item: Following
-            if topicIsFollowing(leftTopic) {
+            // first item: Discover
+            if topicIsDiscover(leftTopic) {
                 return true
             }
-            if topicIsFollowing(rightTopic) {
+            if topicIsDiscover(rightTopic) {
                 return false
             }
 
-            // second item: Discover
-            if topicIsDiscover(leftTopic) {
+            // second item: Following/subscriptions
```

Comment:
Minor: Might want to adjust the comment on line 597 and 613 about the positioning, since Saved is now the third item above Likes.